### ISSUE TITLE
It is not possible to clone or download on windows

### ISSUE THREAD

**Issue by barbosamaatheus** [spgroup/mergedataset#56]
The names of some project directories contain invalid characters for windows and folder names. This causes the error to occur as in the image (https://slack-files.com/T27GALRJ6-F018WEGKN4Q-515de07714) when trying to clone or download the mergedataset.

For my investigations the character "?" it is not allowed on windows.

**Comment by jose** [comment]
Apparently, it doesn't work on MacOS either.  Here's a quick report of my attempts to get this repository.

### Git Clone

```bash
$ git clone https://github.com/spgroup/mergedataset.git
Cloning into 'mergedataset'...
remote: Enumerating objects: 6192, done.
remote: Counting objects: 100% (55/55), done.
remote: Compressing objects: 100% (39/39), done.
error: RPC failed; curl 56 LibreSSL SSL_read: SSL_ERROR_SYSCALL, errno 54
error: 2593 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: index-pack failed

$ git clone https://github.com/spgroup/mergedataset.git
Cloning into 'mergedataset'...
remote: Enumerating objects: 6192, done.
remote: Counting objects: 100% (55/55), done.
remote: Compressing objects: 100% (39/39), done.
error: 4125 bytes of body are still expected | 7.00 KiB/s
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: index-pack failed
```

two attempts and no luck.  I'm currently using `git version 2.30.0`.

### Zip provided by Github

First I tried to download it using Google Chrome.  And when I tried to unzip I got it
```bash
$ unzip mergedataset-master.zip
Archive:  mergedataset-master.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of mergedataset-master.zip or
        mergedataset-master.zip.zip, and cannot find mergedataset-master.zip.ZIP, period.
```

Then I moved to `wget`.
```bash
$ wget https://github.com/spgroup/mergedataset/archive/refs/heads/master.zip
$ unzip master.zip
Archive:  master.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of master.zip or
        master.zip.zip, and cannot find master.zip.ZIP, period.
```
for what's worth the md5sum of the zip file downloaded `wget` is 384360d469cdc17c1515ef313bb3c5a1.


--
Best,
Jose

**Comment by jose** [comment]
> @barbosamaatheus you have already managed to clone it in windows, and explained what to do in the README, right?

The [README file](https://github.com/spgroup/mergedataset#attentions-when-using-windows) does include some documentation for Windows users, yes.

> @jose could you please try again later today so that we are sure this is not a problem with your connection to GitHub (it sounded like that from the error message)?

You're right.  I've just tried to clone it again and it worked.  Thanks.

**Comment by barbosamaatheus** [comment]
Hello everybody.

The error message I got from my problem on Windows explicitly says that the path is invalid in the operating system. As we can see in the image: https://slack-files.com/T27GALRJ6-F018WEGKN4Q-515de07714


That is, the error message I received differs from the message received by @jose.

Since macOS and Linux are based on the same system, I believe the problem that @jose faces is different from mine.

Anyway, I managed to solve this problem by creating a Linux subsystem within my windows, here's the explanation: https://github.com/spgroup/mergedataset#attentions-when-using-windows
